src/debug/debug.cpp

```diff
@@ -1073,6 +1077,7 @@ bool ChangeRegister(char* str)
 }
 
 bool ParseCommand(char* str) {
+	lua_interpreter->RunCommand("print(\"Hello World\\n\")");
```

Comment:
Nice! 😄 

Maybe change it to something more informational, like "Lua says Hello World\n"? Then the oblivious observer knows where this line comes from.

Reply:
I like that 😎